public/javascripts/index.js

```diff
@@ -41,17 +28,39 @@ const Letter = React.createClass({
 	    </div>
 	)
     }
-})
+});
 
-const Alphabet = React.createClass({
-    render() {
+var CapitalAButton = React.createClass({
+    showCapitalA: function(){
+	ReactDOM.render(<Letter />, document.querySelector("#letter"))
```

Comment:
should not use `ReactDOM.render` here, because you won't be able to "unrender" very easily. 

instead use state. `this.setState({showLetter: true})`

then inside your component use `{this.state.showLetter && <Letter />}`